06_puppy/willshen8/syncStore.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import "sync"
+
+type SyncStore struct {
+	syncStore *sync.Map
```

Comment:
You don't have any code that initialises this map, meaning the client must do it. This does not seem like it should be the responsibility of the client. You could add a `NewSyncStore` function that returns an initialised `SyncStore`.